src/main/docs/guide/config/valueAnnotation.adoc

```diff
@@ -6,7 +6,7 @@ Consider the following example:
 
 snippet::io.micronaut.docs.config.value.EngineImpl[tags="imports,class",indent=0,title="@Value Example"]
 
-<1> The `@Value` annotation accepts a string that can have embedded placeholder values (the default value can be provided by specifying a value after the colon `:` character).
+<1> The `@Value` annotation accepts a string that can have embedded placeholder values (the default value can be provided by specifying a value after the colon `:` character). Also try to avoid to set the member visibility to `private`, since this requires micronaut to use reflections. Prefer to use `protected`.
```

Comment:
```suggestion
<1> The `@Value` annotation accepts a string that can have embedded placeholder values (the default value can be provided by specifying a value after the colon `:` character). Also try to avoid setting the member visibility to `private`, since this requires Micronaut Framework to use reflection. Prefer to use `protected`.
```